src/Jira/Issues/Walker.php

```diff
@@ -289,19 +294,18 @@ public function count()
 	/**
 	 * Sets callable.
 	 *
-	 * @param callable $callable Callable.
+	 * @param mixed $callable Callable.
```

Comment:
Why this change? The method accepts only callables. Changing signature would better express this, however, that would also be a BC break.

Reply:
No, you are wrong on both counts
The code clearly accepts anything else besides callable and throws an exeption then
If you wouldnt want to accept those, the guard clause shouldnt be in there.
It is also not a BC break to fix up documentation on those
PHPStan correctly calls out the code for this.

Reply:
We could fake it by saying callable|null or alike
But it would still not be quite the truth.

Reply:
Please:

1. revert the `DocBlock`, so it would use the `callable` data type again
2. add `callable` typehint to the `setDelegate` method (this method)
3. remove the guard clause

---

I think we can safely do this now without causing a BC break because PHP 5.3 is no longer supported.